core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -437,6 +495,67 @@ public PrivateKey getDefaultPrivateKey() throws Exception {
                 "available only for the super tenant.");
     }
 
+    /**
+     * This method loads the private key of a given tenant key store
+     *
+     * @param keyStoreName name of the tenant key store
+     * @param alias        alias of the private key
+     * @return private key corresponding to the alias
```

Comment:
```suggestion
     * This method loads the private key of a given tenant key store.
     *
     * @param keyStoreName Name of the tenant key store.
     * @param alias        Alias of the private key.
     * @return Private key corresponding to the alias.
```

Reply:
Fixed in https://github.com/wso2/carbon-kernel/commit/a0533c76776fd427f270477373ea6951ce2f147f